README.md

```diff
@@ -8,20 +8,19 @@
 [![](https://img.shields.io/pypi/l/sentinelhub.svg)](https://github.com/sentinel-hub/sentinelhub-py/blob/master/LICENSE.md)
 [![Code coverage](https://codecov.io/gh/sentinel-hub/sentinelhub-py/branch/master/graph/badge.svg)](https://codecov.io/gh/sentinel-hub/sentinelhub-py)
 
-# Description
+## Introduction
 
-The **sentinelhub** Python package allows users to make OGC (WMS and WCS)
-web requests to download and process satellite images within your Python
-scripts. It supports Sentinel-2 L1C and L2A, Sentinel-1, Landsat 8, MODIS and DEM data source.
+The **sentinelhub** Python package is the official Python interface for [Sentinel Hub services](https://www.sentinel-hub.com/). It supports most of the services described in the [Sentinel Hub documentation](https://docs.sentinel-hub.com/api/latest/) and any type of [satellite data collections](https://docs.sentinel-hub.com/api/latest/data/), including Sentinel, Landsat, MODIS, DEM, and custom collections produced by users.
 
-The package also supports obtaining data from Amazon Web Service. It can either provide data from public bucket with
-Sentinel-2 L1C imagery or requester pays bucket with Sentinel-2 L2A imagery. If specified the downloaded data can be
-stored in ESA .SAFE format (all types of .SAFE format are supported).
+The package is also a collection of basic tools and utilities for working with geospatial and satellite data. It builds on top of well known packages such as `numpy`, `shapely`, `pyproj`, etc. On the other hand, it serves as a core dependency of [`eo-learn`](https://github.com/sentinel-hub/eo-learn) Python package for creating geospatial data-processing workflows.
```

Comment:
The package **is also** a collection of basic tools and utilities ...
-> The package **also provides** a collection of basic tools and utilities ...

**On the other hand, it serves as** a core dependency of ...
-> **It is also** a core dependency of ...

Reply:
fixed